src/node2vec.jl

```diff
@@ -0,0 +1,136 @@
+Alias = Tuple{SparseVector{Int}, SparseVector{Float64}}
+
+"""
+    Computes node embeddings on graph `g`, as per [1]
+
+Conducts biased random walks on the graph, then computes word embeddings
+by treating those random walks like sentences.
+
+# Arguments
+- `walks_per_node::Int`: Number of walks starting on each node,
+total number of walks is `nv(g) * walks_per_node`
+- `len::Int`: Length of random walks
+- `p::Float64`: Return parameter from [1]
+- `q::Float64`: In-out parameter from [1]
+
+[1] https://cs.stanford.edu/~jure/pubs/node2vec-kdd16.pdf
+"""
+function node2vec(g::AbstractGraph; walks_per_node::Int, len::Int, p::Float64, q::Float64)::Vector{Vector{Int}}
```

Comment:
```suggestion
function node2vec(g::AbstractGraph; walks_per_node::Int=100, len::Int=5, p::Real=0.5, q::Real=0.5)
```

Keyword arguments need default values.